### PR TITLE
Add initial mod_list feature to 1.6.0

### PR DESCRIPTION
**Pull request by scottcwilson**
I didn't add the missing tables feature yet - thought I'd just start here.

### CODE REVIEW

File: zc_install/sql/updates/mysql_upgrade_zencart_160.sql [hunk 1]
```diff
@@ -184,6 +184,8 @@ INSERT INTO admin_pages VALUES ('configWidgets','BOX_CONFIGURATION_WIDGET','FILE
 INSERT INTO configuration (configuration_title, configuration_key, configuration_value, configuration_description, configuration_group_id, sort_order, date_added, use_function, set_function) VALUES
 ('Max Error Logs', 'MAX_ERROR_LOGS', '20', 'Display this number of error logs', @t1, '1', now(), NULL, NULL);
 
+INSERT INTO admin_pages (page_key, language_key, main_page, page_params, menu_key, display_on_menu, sort_order) VALUES ('mod_list', 'BOX_TOOLS_MOD_LIST', 'FILENAME_MOD_LIST', '', 'tools', 'Y', 999);
```
Comment (drbyte) [inline zc_install/sql/updates/mysql_upgrade_zencart_160.sql]: I suggest making this be sort-order 14 to match the main install SQL

File: admin/mod_list.php [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+</table>
+
+<h2><?php echo DB_LIST; ?></h2>
+<?php
+    $new_pages = array(); 
+    $tables_query_raw = "SELECT TABLE_NAME from INFORMATION_SCHEMA.TABLES where TABLE_SCHEMA = '" . DB_DATABASE . "'"; 
+    $tables = $db->Execute($tables_query_raw); 
+    if ($tables->RecordCount() <= 0) { 
+?>
+      <tr><td colspan="3" align="left"><?php echo '<b>' . NO_tables_FOUND . '</b>'; ?></td></tr>
```
Comment (drbyte) [inline admin/mod_list.php]: `NO_tables_FOUND` ... should this be upper-case, and have a lang-define to match it?

**Comment by drbyte** [comment]
I like a lot of what this "tool"/report offers. 

Do you mind reworking the wording and even the names of constants/files to be called something like `System Inspection`, instead of `mod_list`?  ie: `admin/system_inspection.php`, and still on the `Tools` menu.

As for future features on it, I envision not just the "missing tables" idea, but also "how are all my configuration settings different from defaults?" and maybe even a list of all changed/added tax-rates and ot/shipping/payment module settings which differ from default. ... ie: something that lets me very quickly see how to configure a new site identically. (Helpful for debugging, and for upgrading/rebuilding, and more). Just some ideas for future inspiration.

**Comment by scottcwilson** [comment]
No problem, I will take care of the renames shortly.

**Comment by scottcwilson** [comment]
I think that in five years or so you'll be really happy we have this when someone is complaining that their stuff isn't working or that something's wrong.  It's just an easy self-test to run that doesn't require an install (because it's built in).

**Comment by zcwilt** [comment]
:+1:

**Comment by drbyte** [comment]
:+1:

**Pull request merged**